<!Doctype html>
<html class="no-js" lang="">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="x-ua-compatible" content="ie=edge">
        <title>BioX NKU</title>
        <meta name="description" content="">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <link rel="icon" href="img/header-logo.png">
        <!-- Place favicon.ico in the root directory -->
		
		<!-- Google Fonts -->
		<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,500,500i,700,700i" rel="stylesheet">

		<link rel="stylesheet" href="css/bootstrap.min.css">
		<link rel="stylesheet" href="css/slicknav.min.css">
		<link rel="stylesheet" href="css/bootstrap-theme.min.css">
		<link rel="stylesheet" href="css/owl.carousel.min.css">
		<link rel="stylesheet" href="css/owl.theme.default.min.css">
		<link rel="stylesheet" href="css/animate.min.css">
		<link rel="stylesheet" href="css/animate-text.css">
		<link rel="stylesheet" href="css/magnific-popup.css">
        <link rel="stylesheet" href="css/font-awesome.min.css">
		
		<!-- Ruby CSS -->
        <link rel="stylesheet" href="css/normalize.css">
        <link rel="stylesheet" href="style.css">
		<link rel="stylesheet" href="css/responsive.css">
		
		<!-- Color CSS -->
		<!-- <link rel="stylesheet" href="css/color/color7.css"> -->
		<!--<link rel="stylesheet" href="css/color/color1.css">-->
		<!--<link rel="stylesheet" href="css/color/color2.css">-->
		<!--<link rel="stylesheet" href="css/color/color3.css">-->
		<!--<link rel="stylesheet" href="css/color/color4.css">-->
		<!--<link rel="stylesheet" href="css/color/color5.css">-->
		<!--<link rel="stylesheet" href="css/color/color6.css">-->
		<!--<link rel="stylesheet" href="css/color/color8.css">-->
		<link rel="stylesheet" href="css/color/color9.css">

		<link rel="stylesheet" href="#" id="colors">
		
    </head>
    <body>
	
<!-- 		<div class="color-plate ">
			<a class="icon"><i class="fa fa-cog fa-spin"></i></a>
			<h2>Ruby Color</h2>
			<div class="color-head">
				<span class="color1"></span>
				<span class="color2"></span>
				<span class="color3"></span>
				<span class="color4"></span>
				<span class="color5"></span>
				<span class="color6"></span>
				<span class="color7"></span>
				<span class="color8"></span>
				<span class="color9"></span>
			</div>
		</div> -->
		
		<!-- Header Area -->
		<header id="header" class="header">
			<div class="header-inner">
				<div class="container">
					<div class="row">
						<div class="col-md-3 col-sm-2 col-xs-2">
							<div class="logo">
								<a href="index.html"><img src="img/logo.png" alt="logo.png" height="45"></a>
							</div>
						</div>
						<div class="col-md-9 col-sm-10">
							<div class="mobile-menu"></div>
							<nav class="navbar navbar-default">
								<div class="collapse navbar-collapse">
									<ul id="nav" class="nav navbar-nav">
										<li class="current"><a href="#slider">Home</a></li>
										<li><a href="#features">Research</a></li>
										<li><a href="#Services">News</a></li>
										<li><a href="#latest-works">Publications</a></li>
										<li><a href="#blog">Members</a></li>
										<li><a href="#contact">Join us</a></li>
									</ul>
								</div>
							</nav>
						</div>
					</div>
				</div>
			</div>
		</header>
		<!--/End Header Area -->
		
		<!-- Slider Area -->
		<section id="slider">
				<div class="single-slider" style="background-image:url('img/slider1.jpg')">
					<div id="particles-js"></div>
					<div class="container">
						<div class="row">
							<div class="col-md-6 col-md-push-3">
								<div class="text">
									<h1 class="cd-headline clip is-full-width">
										BioX
										<span class="cd-words-wrapper">
											<b class="is-visible">Future</b>
											<b>Interdisciplinary</b>
											<b>Technology</b>
											<b>Mathematics</b>
											<b>Information</b>
											<b>Life Sciences</b>
										</span>
									</h1>
									<p style="font-size: 27px;">Welcome to BioX Group at Nankai University.</br>X for Future, Interdisciplinary, and Technology.</p>
<!-- 									<div class="button-head">
										<div class="button">
											<span></span>
											<a href="#" class="btn">Load More</a>
										</div>
									</div> -->
								</div>
							</div>
						</div>
					</div>
				</div>
		</section>
		<!--/End Slider Area -->
		
		<!-- Research Area -->
		<section id="features" class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-12">
						<div class="section-title">
							<h2>Research</h2>
							<p style="font-size: 21px;">We focus on the interrogation of cell heterogeneity and biological implication in single-cell omics data via mathematical methods and information technologies.</p>
						</div>
					</div>
				</div>
				<div class="row">
					<div class="col-md-6 col-sm-12 col-xs-12 wow fadeInLeft" data-wow-duration="0.8s" data-wow-delay="0.9s">
						<div class="feature-left">
							<img src="img/singlecell.png" alt="#">
						</div>	
					</div>
					<div class="col-md-6 col-sm-12 col-xs-12 wow fadeInRight" data-wow-duration="0.8s" data-wow-delay="0.9s">
						<div class="feature-right">
							<div class="single-right">
								<i class="fa fa-bar-chart" aria-hidden="true"></i>
								<div class="right-content">
									<h3>Missing-data imputation</h3>
									<P>Impute missing data modalities and enhance highly noisy single-cell data via transfer learning.</P>
								</div>
							</div>
							<div class="single-right">
								<i class="fa fa-cubes" aria-hidden="true"></i>
								<div class="right-content">
									<h3>Cell heterogeneity characterization</h3>
									<P>Characterize cell heterogeneity in single-cell data via reference-based unsupervised learning.</P>
								</div>
							</div>
							<div class="single-right">
								<i class="fa fa-magic" aria-hidden="true"></i>
								<div class="right-content">
									<h3>Cell type annotation</h3>
									<P>Annotate cell types in single-cell data automatically via ensemble-based supervised learning.</P>
								</div>
							</div>
							<div class="single-right">
								<i class="fa fa-paper-plane-o" aria-hidden="true"></i>
								<div class="right-content">
									<h3>Cell type-specific pattern elucidation</h3>
									<P>Elucidate cell type-specific patterns in single-cell data via interpretable machine learning.</P>
								</div>
							</div>
						</div>
					</div>
				</div>
			</div>
		</section>
		<!--/End Research Area -->
		
		<!-- News Area -->
		<section id="Services" class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-12">
						<div class="section-title">
							<h2>News</h2>
							</br>
						</div>
					</div>
				</div>
			</div>
		<section id="testimonials" class="section" data-stellar-background-ratio="0.5">
			<section class="bg">
				<div class="container">
					<div class="row">
						<div class="col-md-12 col-sm-12 col-xs-12">
							<div class="owl-carousel testimonial-slider">
								<div class="single-testimonial">
									<h2>MethAgingDB has been accepted by Scientific Data</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for DNA methylation database in aging biology has been accepted for publication in Scientific Data. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>July 2025</span>
								</div>
								<div class="single-testimonial">
									<h2>cpDistiller has been accepted by Nature Communications</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for triple-effect correction in Cell Painting data has been accepted for publication in Nature Communications. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>June 2025</span>
								</div>
								<div class="single-testimonial">
									<h2>Fountain has been accepted by Nature Machine Intelligence</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for rigorous integration of single-cell ATAC-seq data has been accepted for publication in Nature Machine Intelligence. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>May 2025</span>
								</div>
								<div class="single-testimonial">
									<h2>MINGLE has been accepted by Genome Biology</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for automatic cell type annotation in single-cell chromatin accessibility data has been accepted for publication in Genome Biology. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>May 2025</span>
								</div>
								<div class="single-testimonial">
									<h2>MultiKano has been accepted by Protein & Cell</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for automatic cell type annotation in single-cell multi-omics data has been accepted for publication in Protein & Cell. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>December 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>Sijie, Songming, Siyu, Yuhang and Sicheng won National Scholarship</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Sijie, Songming, Siyu, Yuhang and Sicheng won the National Scholarship from Ministry of Education of China. National Scholarship is one of the most outstanding scholarships for graduates and undergraduates in China. Congratulations to Sijie, Songming, Siyu, Yuhang and Sicheng!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>November 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>scBackdoor has been accepted by Cell Discovery</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for backdoor attacks in single-cell pretrained models has been accepted for publication in Cell Discovery. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>November 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>scPRAM has been accepted by Bioinformatics</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for predicting single-cell gene expression perturbation responses has been accepted for publication in Bioinformatics. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>April 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>EpiCarousel has been accepted by Bioinformatics</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for memory- and time-efficient identification of metacells in atlas-level single-cell chromatin accessibility data has been accepted for publication in Bioinformatics. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>April 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>scButterfly has been accepted by Nature Communications</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for versatile single-cell cross-modality translation has been accepted for publication in Nature Communications. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>March 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>CASTLE has been accepted by Nature Computational Science</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for discrete and interpretable embedding of single-cell chromatin accessibility data has been accepted for publication in Nature Computational Science. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>March 2024</span>
								</div>
								<div class="single-testimonial">
									<h2>scCASE has been accepted by Nature Communications</h2>
									<p><i class="fa fa-quote-left" aria-hidden="true"></i><i>Our work for enhancing single-cell chromatin accessibility data has been accepted for publication in Nature Communications. Congratulations!<i class="fa fa-quote-right" aria-hidden="true"></i></i></p>
									<span>February 2024</span>
								</div>
								
							</div>
						</div>
					</div>

				</div>
			</section>
		</section>
		</section>
		<!--/End News Area -->
		
		<!-- Publications Area -->
		<section id="latest-works" class="latest-works section">
			<div class="container">
				<div class="row">
					<div class="col-md-12">
						<div class="section-title">
							<h2>Publications</h2>
						</div>
					</div>
				</div>
				<div class="row">
					<div class="col-md-12 col-sm-12 col-xs-12">
						<div class="works-menu wow fadeInUp" data-wow-duration="0.8s" data-wow-delay="0.3s">
							<ul>
								<li class="active" data-filter="*">All</li>
								<!-- <li data-filter=".preprints">Preprints</li> -->
								<li data-filter=".paper2025">2025</li>
								<li data-filter=".paper2024">2024</li>
								<li data-filter=".paper2023">2023</li>
								<li data-filter=".paper2022">2022</li>
								<li data-filter=".paper2021">2017-2021</li>
							</ul>
						</div>
					</div>
				</div>
				<div class="row">
					<div class="isotop-active">
						<div class="col-md-12 col-sm-12 col-xs-12 paper2021 paper2022 paper2023 paper2024 paper2025">
							<p>† first authors with equal contribution, * corresponding authors</p>
						</div>

						<div class="col-md-12 col-sm-12 col-xs-12 paper2025">
							<p>[45] <u>Shengze Dong†</u>, <u>Songming Tang†</u>, Ding Liu*, <u>Shengquan Chen*</u>. DiTSim: a diffusion-transformers based single-cell ATAC-seq data simulator. <b>Interdisciplinary Sciences: Computational Life Sciences</b>, 2025, Accepted. [<a href="#" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/DiTSim" target="_blank">code</a>]</p>
							<p>[44] <u>Siyu Li†</u>, <u>Songming Tang†</u>, <u>Haocheng Ma</u>, <u>Shengquan Chen*</u>. MethAgingDB: a comprehensive DNA methylation database for aging biology. <b>Scientific Data</b>, 2025, Accepted. [<a href="#" target="_blank">link</a>, <a href="https://methagingdb.biox-nku.cn/" target="_blank">code</a>]</p>
							<p>[43] Chengwei Yan†, <u>Yu Zhang†</u>, Jiuxin Feng†, <u>Heyang Hua</u>, Zhihan Ruan, <u>Zhen Li</u>, <u>Siyu Li</u>, Chaoyang Yan, Pingjing Li, Jian Liu*, <u>Shengquan Chen*</u>. Triple-effect correction for Cell Painting data with contrastive and domain-adversarial learning. <b>Nature Communications</b>, 2025, 16:6886. [<a href="https://doi.org/10.1038/s41467-025-62193-z" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/cpDistiller" target="_blank">code</a>]</p>
							<p>[42] <u>Shuchen Zhu†</u>, <u>Heyang Hua†</u>, <u>Shengquan Chen*</u>. Rigorous integration of single-cell ATAC-seq data using regularized barycentric mapping. <b>Nature Machine Intelligence</b>, 2025, Online. [<a href="https://www.nature.com/articles/s42256-025-01099-3" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/Fountain" target="_blank">code</a>]</p>
							<p>[41] <u>Siyu Li†</u>, <u>Yifan Huang†</u>, <u>Shengquan Chen*</u>. MINGLE: a mutual information-based interpretable framework for automatic cell type annotation in single-cell chromatin accessibility data. <b>Genome Biology</b>, 2025, 26:162. [<a href="https://doi.org/10.1186/s13059-025-03603-9" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/MINGLE" target="_blank">code</a>]</p>
							<p>[40] <u>Xuejian Cui</u>, Qijin Yin, <u>Zijing Gao</u>, <u>Zhen Li</u>, <u>Xiaoyang Chen</u>, Hairong Lv, <u>Shengquan Chen</u>, Qiao Liu, Wanwen Zeng*, Rui Jiang*. CREATE: cell-type-specific cis-regulatory element identification via discrete embedding. <b>Nature Communications</b>, 2025, 16:4607. [<a href="https://www.nature.com/articles/s41467-025-59780-5" target="_blank">link</a>, <a href="https://github.com/cuixj19/CREATE" target="_blank">code</a>]</p>
							<p>[39] <u>Sijie Li†</u>, <u>Heyang Hua†</u>, <u>Shengquan Chen*</u>. Graph neural networks for single-cell omics data: a review of approaches and applications. <b>Briefings in Bioinformatics</b>, 2025, bbaf109. [<a href="https://doi.org/10.1093/bib/bbaf109" target="_blank">link</a>, <a href="https://doi.org/10.1093/bib/bbaf109" target="_blank">code</a>]</p>
							<p>[38] <u>Heyang Hua†</u>, <u>Sijie Li†</u>, <u>Haitian Liang</u>, <u>Shengquan Chen*</u>. Facilitating single-cell chromatin accessibility research with a user-friendly database. <b>Frontiers of Computer Science</b>, 2025, 19(11): 1911920. [<a href="https://doi.org/10.1007/s11704-025-41390-5" target="_blank">link</a>, <a href="#" target="_blank">code</a>]</p>
							<p>[37] <u>Songming Tang</u>, <u>Siyu Li</u>, <u>Shengquan Chen*</u>. Imputing not available values in single-cell DNA methylation data using the median is straightforward and effective. <b>Quantitative Biology</b>, 2025, e70000. [<a href="http://dx.doi.org/10.1002/qub2.70000" target="_blank">link</a>, <a href="https://doi.org/10.5281/zenodo.14601044" target="_blank">code</a>]</p>
							<p>[36] <u>Lan Cao</u>, <u>Wenhao Zhang</u>, Fan Yang, <u>Shengquan Chen</u>, Xiaobing Huang, Feng Zeng*, Ying Wang*. BIOTIC: a Bayesian framework to integrate single-cell multi-omics for transcription factor activity inference and improve identity characterization of cells. <b>Briefings in Bioinformatics</b>, 2025, bbaf013. [<a href="https://doi.org/10.1093/bib/bbaf013" target="_blank">link</a>, <a href="https://github.com/Ying-Lab/BIOTIC" target="_blank">code</a>]</p>
						</div>

						<div class="col-md-12 col-sm-12 col-xs-12 paper2024">
							<p>[35] <u>Siyu Li†</u>, Xinhao Zhuang†, Songbo Jia†, <u>Songming Tang</u>, Liming Yan, <u>Heyang Hua</u>, <u>Yuhang Jia</u>, Xuelin Zhang, Yan Zhang, Qingzhu Yang*, <u>Shengquan Chen*</u>. MultiKano: an automatic cell type annotation tool for single-cell multi-omics data based on Kolmogorov-Arnold network and data augmentation. <b>Protein & Cell</b>, 2024, pwae069. [<a href="https://doi.org/10.1093/procel/pwae069" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/MultiKano" target="_blank">code</a>]</p>
							<p>[34] Qun Jiang, Xiaoyang Chen, Zijing Gao, Jinmeng Jia, <u>Shengquan Chen</u>, Rui Jiang*. CASHeart: a database of single-cells chromatin accessibility for the human heart. <b>Quantitative Biology</b>, 2025, e90. [<a href="http://dx.doi.org/10.1002/qub2.90" target="_blank">link</a>, <a href="http://health.tsinghua.edu.cn/casheartdb" target="_blank">code</a>]</p>
							<p>[33] <u>Sicheng Feng</u>, <u>Siyu Li</u>, Luonan Chen*, <u>Shengquan Chen*</u>. Unveiling potential threats: backdoor attacks in single-cell pretrained models. <b>Cell Discovery</b>, 2024, 10:122. [<a href="https://www.nature.com/articles/s41421-024-00753-1" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/scBackdoor" target="_blank">code</a>]</p>
							<p>[32] <u>Ruike Song†</u>, Xiaofeng Wang†, Jiahao Zhang, <u>Shengquan Chen*</u>, Jianyu Zhou*. GATDE: A graph attention network with diffusion-enhanced protein-protein interaction for cancer classification. <b>Methods</b>, 2024, 231:70-77. [<a href="https://doi.org/10.1016/j.ymeth.2024.09.003" target="_blank">link</a>, <a href="https://github.com/wangxiaofeng64/GATDE" target="_blank">code</a>]</p>
							<p>[31] <u>Heyang Hua†</u>, <u>Wenxin Long†</u>, <u>Yan Pan†</u>, <u>Siyu Li</u>, Jianyu Zhou*, Haixin Wang*, <u>Shengquan Chen*</u>. scCrab: a reference-guided ensemble method for cancer cell identification via Bayesian neural networks. <b>Interdisciplinary Sciences: Computational Life Sciences</b>, 2025, 17:12-26. [<a href="https://doi.org/10.1007/s12539-024-00655-6" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/scCrab" target="_blank">code</a>]</p>
							<p>[30] Zhihan Ruan, Weijun Zhou, Hong Liu, Jinmao Wei, Yichen Pan, Chaoyang Yan, Xiaoyi Wei, Wenting Xiang, Chengwei Yan, <u>Shengquan Chen</u>, Jian Liu*. Precise detection of cell-type-specific domains in spatial transcriptomics. <b>Cell Reports Methods</b>, 2024, 4(8):100841. [<a href="https://doi.org/10.1016/j.crmeth.2024.100841" target="_blank">link</a>, <a href="https://github.com/lyotvincent/Despot" target="_blank">code</a>]</p>
							<p>[29] <u>Qun Jiang†</u>, <u>Shengquan Chen†</u>, <u>Xiaoyang Chen</u>, Rui Jiang*. scPRAM accurately predicts single-cell gene expression perturbation response based on attention mechanism. <b>Bioinformatics</b>, 2024, btae265. [<a href="https://doi.org/10.1093/bioinformatics/btae265" target="_blank">link</a>, <a href="https://github.com/jiang-q19/scPRAM" target="_blank">code</a>]</p>
							<p>[28] <u>Zijing Gao</u>, Rui Jiang, <u>Shengquan Chen*</u>. OpenAnnotateApi: Python and R packages to efficiently annotate and analyze chromatin accessibility of genomic regions. <b>Bioinformatics Advances</b>, 2024, vbae055. [<a href="https://doi.org/10.1093/bioadv/vbae055" target="_blank">link</a>, <a href="https://github.com/ZjGaothu/OpenAnnotatePy" target="_blank">code</a>]</p>
							<p>[27] <u>Sijie Li</u>, <u>Yuxi Li</u>, Yu Sun, Yaru Li, <u>Xiaoyang Chen</u>, <u>Songming Tang</u>, <u>Shengquan Chen*</u>. EpiCarousel: memory- and time-efficient identification of metacells for atlas-level single-cell chromatin accessibility data. <b>Bioinformatics</b>, 2024, btae191. [<a href="https://doi.org/10.1093/bioinformatics/btae191" target="_blank">link</a>, <a href="https://github.com/biox-nku/epicarousel" target="_blank">code</a>]</p>
							<p>[26] <u>Yichuan Cao</u>, <u>Xiamiao Zhao</u>, <u>Songming Tang</u>, <u>Qun Jiang</u>, <u>Sijie Li</u>, <u>Siyu Li</u>, <u>Shengquan Chen*</u>. scButterfly: a versatile single-cell cross-modality translation method via dual-aligned variational autoencoders. <b>Nature Communications</b>, 2024, 15:2973. <b>Editors' Highlights</b>. [<a href="https://www.nature.com/articles/s41467-024-47418-x" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/scButterfly" target="_blank">code</a>]</p>
							<p>[25] <u>Xuejian Cui</u>, <u>Xiaoyang Chen</u>, <u>Zhen Li</u>, <u>Zijing Gao</u>, <u>Shengquan Chen*</u>, Rui Jiang*. Discrete latent embedding of single-cell chromatin accessibility sequencing data for uncovering cell heterogeneity. <b>Nature Computational Science</b>, 2024, 4:346-359. [<a href="https://doi.org/10.1038/s43588-024-00625-4" target="_blank">link</a>, <a href="https://github.com/cuixj19/CASTLE" target="_blank">code</a>]</p>
							<p>[24] <u>Yuhang Jia†</u>, <u>Siyu Li†</u>, Rui Jiang, <u>Shengquan Chen*</u>. Accurate annotation for differentiating and imbalanced cell types in single-cell chromatin accessibility data. <b>IEEE/ACM Transactions on Computational Biology and Bioinformatics</b>, 2024, 21(3):461-471. [<a href="https://ieeexplore.ieee.org/document/10460166" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/CASCADE" target="_blank">code</a>]</p>
							<p>[23] <u>Songming Tang</u>, <u>Xuejian Cui</u>, Rongxiang Wang, <u>Sijie Li</u>, <u>Siyu Li</u>, Xin Huang, <u>Shengquan Chen*</u>. scCASE: accurate and interpretable enhancement for single-cell chromatin accessibility sequencing data. <b>Nature Communications</b>, 2024, 15:1629. <b>Editors' Highlights</b>. [<a href="https://www.nature.com/articles/s41467-024-46045-w" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/scCASE" target="_blank">code</a>]</p>
							<p>[22] Haixin Wang†, <u>Yunhan Wang†</u>, <u>Qun Jiang†</u>, Yan Zhang*, <u>Shengquan Chen*</u>. SCREEN: predicting single-cell gene expression perturbation responses via optimal transport. <b>Frontiers of Computer Science</b>, 2024, 18:183909. [<a href="https://link.springer.com/article/10.1007/s11704-024-31014-9" target="_blank">link</a>, <a href="https://github.com/Califorya/SCREEN" target="_blank">code</a>]</p>
							<p>[21] Keyi Li†, <u>Xiaoyang Chen†</u>, Shuang Song, Lin Hou, <u>Shengquan Chen*</u>, Rui Jiang*. Cofea: correlation-based feature selection for single-cell chromatin accessibility data. <b>Briefings in Bioinformatics</b>, 2024, bbad458. [<a href="https://doi.org/10.1093/bib/bbad458" target="_blank">link</a>, <a href="https://github.com/likeyi19/Cofea" target="_blank">code</a>]</p>
						</div>

						<div class="col-md-12 col-sm-12 col-xs-12 paper2023">
							<p>[20] <u>Wenhao Zhang</u>, Rui Jiang, <u>Shengquan Chen*</u>, Ying Wang*. scIBD: a self-supervised iterative-optimizing model for boosting the detection of heterotypic doublets in single-cell chromatin accessibility data. <b>Genome Biology</b>, 2023, 24:225. [<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03072-y" target="_blank">link</a>, <a href="https://github.com/Ying-Lab/scIBD" target="_blank">code</a>]</p>
							<p>[19] <u>Zijing Gao†</u>, <u>Xiaoyang Chen†</u>, <u>Zhen Li†</u>, <u>Xuejian Cui†</u>, <u>Qun Jiang</u>, Keyi Li, <u>Shengquan Chen*</u>, Rui Jiang*. scEpiTools: a database to comprehensively interrogate analytic tools for single-cell epigenomic data. <b>Journal of Genetics and Genomics</b>, 2023, 51(4):462-465. [<a href="https://doi.org/10.1016/j.jgg.2023.09.011" target="_blank">link</a>, <a href="http://health.tsinghua.edu.cn/scepitools/" target="_blank">tool</a>]</p>
							<p>[18] <u>Zhen Li</u>, <u>Xiaoyang Chen</u>, Xuegong Zhang, Rui Jiang, <u>Shengquan Chen*</u>. Latent feature extraction with a prior-based self-attention framework for spatial transcriptomics. <b>Genome Research</b>, 2023, 33:1757-1773. [<a href="https://genome.cshlp.org/content/early/2023/10/30/gr.277891.123" target="_blank">link</a>, <a href="https://github.com/lizhen18THU/PAST" target="_blank">code</a>]</p>
							<p>[17] Chen Li†, <u>Xiaoyang Chen†</u>, <u>Shengquan Chen</u>,  Rui Jiang*, Xuegong Zhang*. simCAS: an embedding-based method for simulating single-cell chromatin accessibility sequencing data. <b>Bioinformatics</b>, 2023, btad453. [<a href="https://doi.org/10.1093/bioinformatics/btad453" target="_blank">link</a>, <a href="https://github.com/Chen-Li-17/simCAS" target="_blank">code</a>]</p>
							<p>[16] <u>Siyu Li</u>, <u>Songming Tang</u>, <u>Yunchang Wang</u>, <u>Sijie Li</u>, <u>Yuhang Jia</u>, <u>Shengquan Chen*</u>. Accurate cell type annotation for single-cell chromatin accessibility data via contrastive learning and reference guidance. <b>Quantitative Biology</b>, 2023, 12(1):85-99. [<a href="https://onlinelibrary.wiley.com/doi/full/10.1002/qub2.33" target="_blank">link</a>, <a href="https://github.com/biox-nku/rainbow" target="_blank">code</a>]</p>
							<p>[15] Rui Jiang, <u>Zhen Li</u>, <u>Yuhang Jia</u>, <u>Siyu Li</u>, <u>Shengquan Chen*</u>. SINFONIA: scalable identification of spatially variable genes for deciphering spatial domains. <b>Cells</b>, 2023, 12(4):604. [<a href="https://doi.org/10.3390/cells12040604" target="_blank">link</a>, <a href="https://github.com/BioX-NKU/SINFONIA" target="_blank">code</a>]</p>
						</div>
						
						<div class="col-md-12 col-sm-12 col-xs-12 paper2022">
							<p>[14] <u>Shengquan Chen*</u>, Rongxiang Wang, <u>Wenxin Long</u>, Rui Jiang*. ASTER: accurately estimating the number of cell types in single-cell chromatin accessibility data. <b>Bioinformatics</b>, 2022, btac842. [<a href="https://doi.org/10.1093/bioinformatics/btac842" target="_blank">link</a>, <a href="https://github.com/biox-nku/aster" target="_blank">code</a>]</p>
							<p>[13] Zheng Zhang, <u>Shengquan Chen</u>, Zhixiang Lin*. RefTM: reference-guided topic modeling of single-cell chromatin accessibility data. <b>Briefings in Bioinformatics</b>, 2022, bbac540. [<a href="https://doi.org/10.1093/bib/bbac540" target="_blank">link</a>, <a href="https://github.com/cuhklinlab/RefTM" target="_blank">code</a>]</p>
							<p>[12] <u>Xiaoyang Chen†</u>, <u>Shengquan Chen†</u>, Shuang Song, <u>Zijing Gao</u>, Lin Hou, Xuegong Zhang, Hairong Lv, Rui Jiang*. Cell type annotation of single-cell chromatin accessibility data via supervised Bayesian embedding. <b>Nature Machine Intelligence</b>, 2022, 4:116-126. [<a href="https://doi.org/10.1038/s42256-021-00432-w" target="_blank">link</a>, <a href="https://github.com/xy-chen16/EpiAnno" target="_blank">code</a>]</p>
							<p>[11] Xi Xi, Haochen Li, <u>Shengquan Chen</u>, Tingting Lv, Tianxing Ma, Rui Jiang, Ping Zhang, Wing Hung Wong*, Xuegong Zhang*. Unfolding the genotype-to-phenotype black box of cardiovascular diseases through cross-scale modeling. <b>iScience</b>, 2022, 25(8):104790. [<a href="https://www.sciencedirect.com/science/article/pii/S2589004222010628" target="_blank">link</a>, <a href="https://github.com/xixi-cathy/GRPath" target="_blank">code</a>]</p>
						</div>

						<div class="col-md-12 col-sm-12 col-xs-12 paper2021">
							<p>[10] <u>Shengquan Chen†</u>, Guanao Yan†, Wenyu Zhang, Jinzhao Li, Rui Jiang*, Zhixiang Lin*. RA3 is a reference-guided approach for epigenetic characterization of single cells. <b>Nature Communications</b>, 2021, 12:2177. [<a href="https://www.nature.com/articles/s41467-021-22495-4" target="_blank">link</a>, <a href="https://github.com/cuhklinlab/RA3" target="_blank">code</a>]</p>
							<p>[9] <u>Shengquan Chen</u>, Qiao Liu, <u>Xuejian Cui</u>, Zhanying Feng, Chunquan Li, Xiaowo Wang, Xuegong Zhang, Yong Wang, Rui Jiang*. OpenAnnotate: a web server to annotate the chromatin accessibility of genomic regions. <b>Nucleic Acids Research</b>, 2021, 49(W1):W483-W490. [<a href="https://doi.org/10.1093/nar/gkab337" target="_blank">link</a>, <a href="http://health.tsinghua.edu.cn/openannotate/" target="_blank">tool</a>]</p>
							<p>[8] Wanwen Zeng†, <u>Shengquan Chen†</u>, <u>Xuejian Cui†</u>, <u>Xiaoyang Chen</u>, <u>Zijing Gao</u>, Rui Jiang*. SilencerDB: a comprehensive database of silencers. <b>Nucleic Acids Research</b>, 2021, 49(D1):D221-D228. [<a href="https://academic.oup.com/nar/article-abstract/49/D1/D221/5921294" target="_blank">link</a>, <a href="http://health.tsinghua.edu.cn/silencerdb/" target="_blank">tool</a>]</p>
							<p>[7] <u>Shengquan Chen</u>, Mingxin Gan, Hairong Lv, Rui Jiang*. DeepCAPE: a deep convolutional neural network for the accurate prediction of enhancers. <b>Genomics, Proteomics & Bioinformatics</b>, 2021, 19(4):565-577. [<a href="https://www.sciencedirect.com/science/article/pii/S1672022921000127" target="_blank">link</a>, <a href="https://github.com/ShengquanChen/DeepCAPE" target="_blank">code</a>]</p>
							<p>[6] <u>Shengquan Chen</u>, Boheng Zhang, <u>Xiaoyang Chen</u>, Xuegong Zhang, Rui Jiang*. stPlus: a reference-based method for the accurate enhancement of spatial transcriptomics. <b>ISMB/ECCB 2021, Bioinformatics</b>, 2021, 37(S1):i299-i307. [<a href="https://doi.org/10.1093/bioinformatics/btab298" target="_blank">link</a>, <a href="http://health.tsinghua.edu.cn/software/stPlus/" target="_blank">tool</a>]</p>
							<p>[5] Qiao Liu, <u>Shengquan Chen</u>, Rui Jiang*, Wing Hong Wong*. Simultaneous deep generative modeling and clustering of single cell genomic data. <b>Nature Machine Intelligence</b>, 2021, 3:536-544. [<a href="https://www.nature.com/articles/s42256-021-00333-y" target="_blank">link</a>, <a href="https://github.com/kimmo1019/scDEC" target="_blank">code</a>]</p>
							<p>[4] <u>Xiaoyang Chen</u>, <u>Shengquan Chen</u>, Rui Jiang*. EnClaSC: A novel ensemble approach for accurate and robust cell-type classification of single-cell transcriptomes. <b>BMC Bioinformatics</b>, 2020, 21(13):1-16. [<a href="https://link.springer.com/article/10.1186/s12859-020-03679-z" target="_blank">link</a>, <a href="https://github.com/xy-chen16/EnClaSC" target="_blank">code</a>]</p>
							<p>[3] <u>Shengquan Chen</u>, Kui Hua, Hongfei Cui, Rui Jiang*. VPAC: Variational projection for accurate clustering of single-cell transcriptomic data. <b>BMC Bioinformatics</b>, 2019, 20(7):139-151. [<a href="https://link.springer.com/article/10.1186/s12859-019-2742-4" target="_blank">link</a>, <a href="https://github.com/ShengquanChen/VPAC" target="_blank">code</a>]</p>
							<p>[2] Shaoming Song, Hongfei Cui, <u>Shengquan Chen</u>, Qiao Liu, Rui Jiang*. EpiFIT: functional interpretation of transcription factors based on combination of sequence and epigenetic information. <b>Quantitative Biology</b>, 2019, 7(3):233-243. [<a href="https://link.springer.com/article/10.1007/s40484-019-0175-8" target="_blank">link</a>, <a href="https://health.tsinghua.edu.cn/openness/EpiFIT/" target="_blank">tool</a>]</p>
							<p>[1] Xu Min†, Wanwen Zeng†, <u>Shengquan Chen</u>, Ning Chen, Ting Chen, Rui Jiang*. Predicting enhancers with deep convolutional neural networks. <b>BMC Bioinformatics</b>, 2017, 18(13):478. [<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1878-3" target="_blank">link</a>, <a href="https://github.com/minxueric/DeepEnhancer" target="_blank">code</a>]</p>
						</div>
					</div>
				</div>
				<div class="row">
					<div class="col-md-12">
						<div class="button-head">
							<div class="button">
								<span></span>
								<a href="https://scholar.google.com/citations?user=z5p5bjIAAAAJ&hl=en" class="btn" target="_blank">Load More</a>
							</div>
						</div>
					</div>
				</div>
			</div>
		</section>
		<!--/End Publications Area -->
		
		<!-- Members -->
		<section id="blog" class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-12">
						<div class="section-title">
							<h2>Members</h2>
						</div>
					</div>
				</div>
				<div class="row" style="margin-top: 20px;">
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.3s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/chenshengquan.jpg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Principal Investigator</h3>
								<h2>Shengquan Chen</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Professor, Nankai U. (2025.1-)</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Associate Prof., Nankai U. (2022.1-2024.12)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>Ph.D., Tsinghua U. (2017.9-2021.12)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Eng., Xiamen U. (2013.9-2017.7)</p>
								<a href="https://my.nankai.edu.cn/sms/csq/list.htm" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.6s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/lihan.jpg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Assistant Prof.</h3>
								<h2>Han Li</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Assistant Prof., Nankai U. (2024.7-)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>Ph.D., Tsinghua U. (2019.9-2024.6)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Eng., Jilin U. (2015.9-2019.7)</p>
								<a href="https://math.nankai.edu.cn/2024/0812/c34830a548728/page.htm" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.9s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/huaheyang.jpg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Ph.D. Student</h3>
								<h2>Heyang Hua</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Ph.D. Student, Nankai U. (2025.9-)</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Visiting Student, Perelman School of Medicine, University of Pennsylvania (2025.2-2025.8)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., Chern Class of Nankai Mathematics (2021.9-2025.7)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
				</div>


				<div class="row" style="margin-top: 20px;">
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.3s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/hulingyi.jpg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Ph.D. Student</h3>
								<h2>Lingyi Hu</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Ph.D. Student, Nankai U. (2025.9-)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>M.Phil., Hunan U. (2022.9-2025.7)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., CAUC. (2018.9-2022.7)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.6s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/tangsongming.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Ph.D. Student</h3>
								<h2>Songming Tang</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Ph.D. Student, Nankai U. (2025.9-)</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>M.Phil. Student, Nankai U. (2023.9-)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., SCUT. (2019.9-2023.7)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.9s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/caolan.jpg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Joint Ph.D. Student with Xiamen Univ.</h3>
								<h2>Lan Cao</h2>
								<p style="text-align:center;">School of Aerospace Engineering</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Ph.D. Student, Xiamen U. (2024.9-)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>M.Phil., Xiamen U. (2021.9-2024.7)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Eng., Xiamen U. (2016.9-2020.7)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
				</div>

				<div class="row" style="margin-top: 20px;">
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.3s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/zhangyu.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">M.Phil. Student</h3>
								<h2>Yu Zhang</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>M.Phil. Student, Nankai U. (2023.9-)</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Intern, ByteDance. (2024.11-)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., NWAFU. (2019.9-2023.7)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.6s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/lisiyu.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">M.Phil. Student</h3>
								<h2>Siyu Li</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>M.Phil. Student, Nankai U. (2024.9-)</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Visiting Student, Department of Statistics, The Chinese University of Hong Kong. (2024.1-2024.6)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., Nankai U. (2020.9-2024.7)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.9s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/jiayuhang.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">M.Phil. Student</h3>
								<h2>Yuhang Jia</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>M.Phil. Student, Nankai U. (2024.9-)</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Visiting Student, Shenzhen Institute of Advanced Technology, CAS. (2023.10-2024.6)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., Nankai U. (2020.9-2024.7)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
				</div>


				<div class="row" style="margin-top: 20px;">
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.3s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/liyongkang.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">M.Phil. Student</h3>
								<h2>Yongkang Li</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>M.Phil. Student, Nankai U. (2024.9-)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., Chern Class of Nankai Mathematics (2020.9-2024.7)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.6s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/qinghuangpei.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">M.Phil. Student</h3>
								<h2>Huangpei Qing</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>M.Phil. Student, Nankai U. (2025.9-)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., Chern Class of Nankai Mathematics (2021.9-2025.7)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.9s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/zouziyi.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">M.Phil. Student</h3>
								<h2>Ziyi Zou</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>M.Phil. Student, Nankai U. (2025.9-)</p>
								<p><i class="fa fa-graduation-cap" aria-hidden="true"></i>B.Sc., DUT. (2021.9-2025.7)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
				</div>


				<div class="row" style="margin-top: 20px;">
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.3s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/gukeju.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">B.Sc. Student</h3>
								<h2>Keju Gu</h2>
								<p style="text-align:center;">College of Life Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Next position: Ph.D. Student, Nankai U.</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>B.Sc., Nankai U. (2022.9-)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.6s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/lianghaitian.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Chern Class of Nankai Mathematics</h3>
								<h2>Haitian Liang</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>Visiting Student, Westlake University (2025.7-2025.9)</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>B.Sc., Nankai U. (2023.9-)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.9s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/linyiming.jpeg" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">B.Sc. Student</h3>
								<h2>Yiming Lin</h2>
								<p style="text-align:center;">School of Mathematical Sciences</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>B.Sc., Nankai U. (2023.9-)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
				</div>

				<div class="row" style="margin-top: 20px;">
					<div class="col-md-4 col-sm-12 col-xs-12 wow fadeIn" data-wow-duration="0.8s" data-wow-delay="0.3s">
						<div class="single-news">
							<div class="news-head" align="center">
								<img src="img/kaikai.png" alt="#">
							</div>
							<div class="news-body">
								<h3 class="date">Encourager</h3>
								<h2>Kaikai</h2>
								<p style="text-align:center;">School of Happiness</p>
								<p><i class="fa fa-leanpub" aria-hidden="true"></i>B.Sc., Home U. (2022.9-)</p>
								<a href="#" class="btn">Read More<i class="fa fa-caret-right" aria-hidden="true"></i></a>
							</div>
						</div>
					</div>
				</div>
			</div>
		</section>
		<!-- /End Members Area -->

		<!-- Gallery Area -->
		<section id="gallery" class="section">
		    <div class="container">
		        <div class="row">
		            <div class="col-md-12">
		                <div class="section-title">
		                    <h2>Gallery</h2>
		                </div>
		            </div>
		        </div>
		        <div class="row" style="margin-top: 20px;">
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab1.jpg" class="gallery-item">
		                    <img src="img/lab1_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab2.jpg" class="gallery-item">
		                    <img src="img/lab2_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab3.jpg" class="gallery-item">
		                    <img src="img/lab3_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab4.jpg" class="gallery-item">
		                    <img src="img/lab4_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		        </div>

		        <div class="row" style="margin-top: 20px;">
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab5.jpg" class="gallery-item">
		                    <img src="img/lab5_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab6.jpg" class="gallery-item">
		                    <img src="img/lab6_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab7.jpg" class="gallery-item">
		                    <img src="img/lab7_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab8.jpg" class="gallery-item">
		                    <img src="img/lab8_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		        </div>

		        <div class="row" style="margin-top: 20px;">
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab9.jpg" class="gallery-item">
		                    <img src="img/lab9_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab10.jpg" class="gallery-item">
		                    <img src="img/lab10_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab11.jpg" class="gallery-item">
		                    <img src="img/lab11_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab12.jpg" class="gallery-item">
		                    <img src="img/lab12_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		        </div>

		        <div class="row" style="margin-top: 20px;">
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab13.jpg" class="gallery-item">
		                    <img src="img/lab13_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab14.jpg" class="gallery-item">
		                    <img src="img/lab14_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab15.jpg" class="gallery-item">
		                    <img src="img/lab15_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab16.jpg" class="gallery-item">
		                    <img src="img/lab16_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		        </div>

		        <div class="row" style="margin-top: 20px;">
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab17.jpg" class="gallery-item">
		                    <img src="img/lab17_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab18.jpg" class="gallery-item">
		                    <img src="img/lab18_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab19.jpg" class="gallery-item">
		                    <img src="img/lab19_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		            <div class="col-md-3 col-sm-6 col-xs-12">
		                <a href="img/lab20.jpg" class="gallery-item">
		                    <img src="img/lab20_thumbnail.jpg" class="img-thumbnail">
		                </a>
		            </div>
		        </div>
		    </div>
		</section>
		
		<!-- Alumni Area -->
		<section id="alumni" class="latest-works section">
			<div class="container">
				<div class="row">
					<div class="col-md-12">
						<div class="section-title">
							<h2>Alumni</h2>
						</div>
					</div>
				</div>
				<div class="row" style="margin-top: 20px;">
					<div class="col-md-12 col-sm-12 col-xs-12">
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Yichuan Cao, Undergraduate student, 2022.7-2023.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Academy of Mathematics and Systems Science, Chinese Academy of Sciences</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Xiamiao Zhao, Undergraduate student, 2022.7-2023.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in School of Mathematical Sciences, Tsinghua University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Yunhan Wang, Undergraduate student, 2022.11-2023.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: MS in School of Statistics, Renmin University of China</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Yuxi Li, Undergraduate student, 2022.7-2023.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: MS in School of Cyber Science and Engineering, Huazhong University of Science and Technology</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Wenhao Zhang, Postgraduate student, 2022.7-2023.10</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in School of Aerospace Engineering, Xiamen University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Xiaoyang Chen, Postgraduate student, 2022.2-2024.1</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Department of Automation, Tsinghua University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Zijing Gao, Postgraduate student, 2023.1-2024.4</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Department of Automation, Tsinghua University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Xuejian Cui, Postgraduate student, 2022.8-2024.5</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Department of Automation, Tsinghua University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Qun Jiang, Postgraduate student, 2023.5-2024.5</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Department of Automation, Tsinghua University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Yuhang Jia, Undergraduate student, 2022.4-2024.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: MS in School of Mathematical Sciences, Nankai University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Siyu Li, Undergraduate student, 2022.4-2024.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: MS in School of Mathematical Sciences, Nankai University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Wenxin Long, Undergraduate student, 2022.5-2024.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Statistics, Eberly College of Science, Penn State University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Shuchen Zhu, Undergraduate student, 2022.10-2024.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Academy for Advanced Interdisciplinary Studies, Peking University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Yifan Xu, Undergraduate student, 2023.9-2024.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: MS in Financial Engineering, New York University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Hongyang Pang, Undergraduate student, 2023.12-2024.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: MS in Department of Statistics and Data Science, Northwestern University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Rufeng Li, Postgraduate student, 2024.3-2024.9</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in School of Basic Medical Sciences, Xi’an Jiaotong University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Yufei Yao, Postgraduate student, 2024.3-2024.10</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in School of Basic Medical Sciences, Xi’an Jiaotong University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Zhen Li, Postgraduate student, 2022.11-2024.11</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Department of Automation, Tsinghua University</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Yan Pan, Postgraduate student, 2023.5-2024.11</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Perelman School of Medicine, University of Pennsylvania</p>
						<p><i class="fa fa-paper-plane-o" aria-hidden="true"></i> Sijie Li, Postgraduate student, 2022.9-2025.7</br>&nbsp;&nbsp;&nbsp;&nbsp; Next position: PhD in Institute of Science and Technology for Brain-inspired Intelligence, Fudan University</p>

					</div>						
				</div>
			</div>
		</section>
		<!--/End Alumni Area -->

		<!--/ Contact Area -->
		<section id="contact" class="contact section" data-stellar-background-ratio="0.5">
				<div class="container">
					<div class="row">
						<div class="contact-head">
							<div class="col-md-4 col-sm-4 col-xs-12">
								<div class="contact-left">
									<h3>Join us now</h3>
									<p class="p1">We are now looking for self-motivated graduate/undergraduate students.</br>If you are interested in and want to join BioX at NKU, feel free to contact us.</p>
									
								</div>
							</div>
							<div class="col-md-8 col-sm-8 col-xs-12">
								<div class="contact-right">
									<h3>Contact information</h3>
									<p class="p1">Dr. Shengquan Chen</p>
									<p>shengquan[dot]chenl[at]gmail[dot]com</p>
									<p>Room 411, Department of Informatics Theory and Data Science</p>
									<p>School of Mathematical Sciences, Nankai University, Tianjin, China</p>
								</div>
							</div>
						</div>
					</div>
				</div>
		</section>
		<!--/End Contact Area -->
		
		<!-- Footer Area -->
		<footer id="footer" class="footer section">
			<div class="copyright">
				<div class="container">
					<div class="row">
						<div class="col-md-12 col-sm-12 col-xs-12">
							<div class="copyright-content">
								<p>Copyright &copy; 2022. BioX at NKU. All rights reserved.</p>
							</div>
						</div>
					</div>
				</div>
			</div>
		</footer>
		<!--/End Footer Area -->
			
        <script src="js/jquery.min.js"></script>
        <script src="js/bootstrap.min.js"></script>
		<script src="js/colors.js"></script>
        <script src="js/jquery.nav.js"></script>
        <script src="js/jquery.scrollUp.min.js"></script>
        <script src="js/jquery.slicknav.min.js"></script>
		<script src="js/waypoints.min.js"></script>
        <script src="js/owl.carousel.min.js"></script>
		<script src="js/isotope.pkgd.min.js"></script>
		<script src="js/wow.min.js"></script>
		<script src="js/jquery.stellar.min.js"></script>
		<script src="js/jquery.magnific-popup.min.js"></script>
		<script src="js/animate-text.js"></script>
		<script src="js/particles.min.js"></script>
		<script src="js/particle-code.js"></script>
		<script src="js/jquery.counterup.min.js"></script>
		
        <script type="text/javascript" src="js/gmaps.min.js"></script>
        <script src="js/main.js"></script>
    </body>
</html>